src/tools/jobs/send-email-prefets.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -l
+
+cd ${APP_HOME}
+
+echo "Sending email to prefets: START\n"
+node $ROOT/src/tools/admin/prefets/account
```

Comment:
Pas sûr du $ROOT. Moi je n'ai rien mis  : 

node src/tools/scripts/migrate-pg-to-mongo.js

(au dessus il y a le cd ${APP_HOME} )

Reply:
bien vu, copier/coller trop rapide